### PR TITLE
Modify Machine Learning alarms

### PR DESCRIPTION
**Pull request by bryan-robitaille**
This PR removes the machine learning generated alarms in favour for static alarms.

Response times now generate alarms if average is above 1 second for two 1 minute periods out of 5 minutes.

Availability now generates a warning is health checks are below for two 1 minute periods out of 5 minutes.
NEW Availability now generates a critical alarm if health checks are 0 for two 1 minutes periods out of 5 minutes.

### CODE REVIEW

File: config/terraform/aws/cloudwatch.tf [hunk 1]
```diff
@@ -290,21 +284,37 @@ resource "aws_cloudwatch_log_metric_filter" "service_availability" {
 
 resource "aws_cloudwatch_metric_alarm" "service_availability_warn" {
   alarm_name          = "ServiceAvailabilityWarn"
-  comparison_operator = "LessThanLowerThreshold"
+  comparison_operator = "LessThanThreshold"
   evaluation_periods  = "5"
   datapoints_to_alarm = "2"
-  threshold_metric_id = "e1"
-  alarm_description   = "COVID Alert Portal Warning - No status checks detected.  Check COVID Alert Portal to ensure site is operational."
+  threshold_metric_id = "18"
```
Comment (pcraig3) [inline config/terraform/aws/cloudwatch.tf:290]: Is this right? If we want a number here, don't we rename this to `threshold`, not `threshold_metric_id`? Or is 18 an ID?

File: config/terraform/aws/cloudwatch.tf [hunk 2]
```diff
@@ -290,21 +284,37 @@ resource "aws_cloudwatch_log_metric_filter" "service_availability" {
...
+    }
   }
+}
+
+resource "aws_cloudwatch_metric_alarm" "service_availability_critical" {
+  alarm_name          = "ServiceAvailabilityCritical"
+  comparison_operator = "LessThanThreshold"
+  evaluation_periods  = "5"
+  datapoints_to_alarm = "2"
+  threshold_metric_id = "1"
```
Comment (pcraig3) [inline config/terraform/aws/cloudwatch.tf:313]: Similar to above, should this be `threshold`?

**Review by pcraig3 — COMMENTED** [review]
A couple small comments just.

File: config/terraform/aws/cloudwatch.tf (hunk 1 above)
Comment (bryan-robitaille) [reply config/terraform/aws/cloudwatch.tf]: yup, typo

File: config/terraform/aws/cloudwatch.tf (hunk 2 above)
Comment (bryan-robitaille) [reply config/terraform/aws/cloudwatch.tf]: yup, typo

**Review by pcraig3 — APPROVED** [review]
Looks good 👍

**Pull request merged**